Components/Queues/QueueManager.php

```diff
@@ -131,6 +131,17 @@ public function getQueue(string $name, int $flag = AMQP_DURABLE, array $argument
         return $queue;
     }
 
+    /**
+     * @param string $queueName
+     * @param string $message
+     * @param array $headers
+     * @return bool
+     * @throws \AMQPChannelException
+     * @throws \AMQPConnectionException
+     * @throws \AMQPException
+     * @throws \AMQPExchangeException
+     * @throws \AMQPQueueException
+     */
```

Comment:
Is still needed this class if you deleted everything that was related to the old Amqp implementation?

Reply:
The file was removed.